optaplanner-core/src/main/resources/solver.xsd

```diff
@@ -0,0 +1,1622 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Sure it would, on the other hand, the XSD generation takes ~17 seconds on my laptop.

Options:
- run always by default 
- run always only in the CI
- run only with release (there is going to be some automation for publishing the XSD)

Reply:
Hmm, I agree that 17 seconds is a bit too much. 

In that case, maybe we should have two tests. This one, plus another one that's only activated if the generated XSD is present. The question of when to generate the XSD is a good one - I'd argue that if it's not tested regularly and frequently, it's a problem.